pkg/api/controller_test.go

```diff
@@ -3647,20 +3647,21 @@ func TestLogin(t *testing.T) {
 	claims, err := auth.VerifyToken(deps.authService.SecretStore().SharedSecret(), res.Token)
 	testutil.Must(t, err)
 	resultExpiry := swag.Int64Value(res.TokenExpiration)
-	if claims.ExpiresAt != resultExpiry {
-		t.Errorf("token expiry (%d) not equal to expiry result (%d)", claims.ExpiresAt, resultExpiry)
+	if claims.ExpiresAt.Unix() != resultExpiry {
+		t.Errorf("token expiry (%d) not equal to expiry result (%d)", claims.ExpiresAt.Time.Unix(), resultExpiry)
```

Comment:
Nit: Consider converting in the opposite direction -- make the expected a time.Time, rather than checking that the actual converts to the right Unix time.